PoC-updater/updater.go

```diff
@@ -73,26 +91,65 @@ func process(storage Storage) {
 	c := types.ClusterEntry{
 		OrgID:       testedOrgID,
 		ClusterName: testedCluster,
+		UpdatedAt:   timestamp,
 	}
 
-	// read older reports for a cluster
-	reported, err := storage.ReadLastNNotificationRecords(c, 10)
+	// try to read older report for a cluster
+	reported, err := storage.ReadLastNNotificationRecords(c, 1)
```

Comment:
So the idea is that this will be how we process all message right? Which is why now we are only getting 1 item from the reported table?

Reply:
`reported` table contains already reported/notified/ignored reports. Right now for the MVP I think it is ok to compare just with the latest report, yes